helper-scripts/sort-versions.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env node */
```

Comment:
The filename `sort-versions.js` does not imply that it "only" sorts compat data by `version_added`.